libs/payments/ui/src/lib/client/components/SelectTaxLocation/index.tsx

```diff
@@ -421,6 +421,16 @@ const Expanded = ({
                   invalidPostalCode: false,
                 }));
               }}
+              onInvalid={(e: React.FormEvent<HTMLInputElement>) => {
+                if (e.currentTarget.validity.patternMismatch) {
```

Comment:
I believe that many of these subfields can be undefined. Can you update this with optional chaining? e.g. `e?.currentTarget?.{...}`